### DISCUSSION TITLE
Sprawy organizacyjne

### DISCUSSION THREAD

**Discussion by hawkeye116477** [MajkiIT/polish-ads-filter#19866] [General]
> co się ostatnio dzieje? Czemu ignorujecie ostatnio ludzi? Ale tak szczerze? Zawsze pomagaliście, a teraz jeden robi wojnę z ludźmi, drugi ignoruje wszystko i tak dalej. Czemu nie dacie tego reopen, jeżeli autor tematu o to prosi? To nie jest miłe, że ignorujecie i nie mówicie ani słowa w temacie.

@seba2282 Tyle, że to `critc` zamknął zgłoszenia, ani ja ani Krystian nie jesteśmy tutaj adminami...
Tak w ogóle to critic już zwracał mu uwagę, żeby otwierał odpowiednio zgłoszenia, czyli z etykietą, a ten robił dalej tak samo, czyli olał sprawę, więc critc zamknął. Nie wiem czy ty byś też pomagał takiej osobie, która olewa coś, gdy ją się o to prosi.

A tak w ogóle, to Pan Felix blokuje wiele osób, więc one nawet nie mogą pisać w jego wątkach, tak samo zresztą oni jego z powodu jego dość specyficznego zachowania.

**Comment by seba2282** [comment]
> > co się ostatnio dzieje? Czemu ignorujecie ostatnio ludzi? Ale tak szczerze? Zawsze pomagaliście, a teraz jeden robi wojnę z ludźmi, drugi ignoruje wszystko i tak dalej. Czemu nie dacie tego reopen, jeżeli autor tematu o to prosi? To nie jest miłe, że ignorujecie i nie mówicie ani słowa w temacie.
> 
> @seba2282 Tyle, że to `critc` zamknął zgłoszenia, ani ja ani Krystian nie jesteśmy tutaj adminami...
> Tak w ogóle to critic już zwracał mu uwagę, żeby otwierał odpowiednio zgłoszenia, czyli z etykietą, a ten robił dalej tak samo, czyli olał sprawę, więc critc zamknął. Nie wiem czy ty byś też pomagał takiej osobie, która olewa coś, gdy ją się o to prosi.
> 
> A tak w ogóle, to Pan Felix blokuje wiele osób, więc one nawet nie mogą pisać w jego wątkach, tak samo zresztą oni jego z powodu jego dość specyficznego zachowania.



Tylko, że wchodzi ostatnio @xxcriticxx I on zamyka. Nawet mi on nie chce udzielić informacji pod moimi komentarzami, co zostało dodane. Moglibyście jakoś mówić przed zamknięciem numer id do reguł albo napisać w komentarzu jak @MajkiIT napisał o jakimś bs item dla przykładu? Ja też oczekuję jakiś informacji zwrotnej. Nic więcej. Ja szanuję Was, wy mnie i byłoby na prawdę miło wiedzieć, jaka reguła/reguły zamykają temat i są zatwierdzone. Nie wiem do końca co z tym Felix, ale że jestem dobry, chciałem trochę rozluźnić sytuację, bo nie potrzebnie jest to całe napięcie :)

**Reply by hawkeye116477** [reply]
> jaka reguła/reguły zamykają temat

Przy każdym Pull Request masz napisane które wątki on zamyka, np tutaj https://github.com/MajkiIT/polish-ads-filter/pull/19863. A konkretne reguły możesz znaleźć w commitach PR.

**Reply by seba2282** [reply]
Widzę ten całą zlepkę tematów pozamykanych tutaj. Ale jeszcze do końca nie umiem się odnaleźć w tym. Wiesz mi :)

**Reply by hawkeye116477** [reply]
Po prostu klikasz w [Files changed](https://github.com/MajkiIT/polish-ads-filter/pull/19863/files) i tam masz reguły, po prostu musisz poszukać, bo też nikt pojedynczo nie dodaje jednej rzeczy, jeżeli jest mnóstwo spraw do rozwiązania.

**Reply by seba2282** [reply]
Właśnie tak patrzę teraz na te merged. :)

**Reply by seba2282** [reply]
Więc myślę, że @MajkiIT powinien teraz wejść i już zamknąć sprawę Felix. Jeżeli tak sprawa wygląda, to nie ma co ciągnąć dalej. I cieszę się, że stworzyłeś tą dyskusję, @hawkeye116477. Przynajmniej wiem, że nie jestem ignorowany tutaj. :)

**Reply by seba2282** [reply]
Więc @xxcriticxx dał mu odpowiednią etykietkę i zamknął temat. Ale te jego słowa, tego Felixa były trochę lekceważące. Takie mam odczucie.

**Reply by hawkeye116477** [reply]
Felixa znają osoby nie tylko te co odwiedziały repo Majkela, ale również forum `https://mozillapl.org/forum/index.php`, gdzie dostał :medal_sports: od moderatora w nagrodę za zasługi. Odwiedzał też chyba repo EasyList i gorhilla, oczywiście pod innym nickiem, jako garry czy jakoś tak, bo on co jakiś czas tworzy nowe konta pod innym IP, bo jak on to mówi - życzliwie na niego donoszą.

**Reply by seba2282** [reply]
@hawkeye116477, więc Felix ma bogaty życiorys. 😅😅😅😅😅😅😅 
Cóż gdybym wiedział wcześniej, nie próbowałbym mu pomóc. Ale cóż, mój błąd.
